jdcarey128/Crate
# Update/user be fe

## What's this PR do?
- Connects `Profile` and `ProfileForm` to the backend
- Adds user details to `Profile` 
- Fixes `onUpload` function in `ProfileForm`
- Completes functionality for `onSubmit` in `ProfileForm`

## Relevant tickets
- Closes #78, #55, #54, #52, #49, #56, #83, #87

## Where should the reviewer start?
-  For FE: review `Profile` and `ProfileForm`

## How should this be manually tested?
- Test in browser by logging in and clicking `Edit Profile`, add a picture, change description, shipping address, or email. 
- Changes should be reflected on Profile page

## Any background context you want to provide?
- n/a

## Screenshots/GIFs
<img width="649" alt="Screen Shot 2021-02-12 at 9 43 15 PM" src="https://user-images.githubusercontent.com/68157759/107839652-55a1bc80-6d7b-11eb-9e61-88d272aad16b.png">


## Questions:
- n/a
